spec/iex/client_spec.rb

```diff
@@ -135,4 +135,36 @@
       end
     end
   end
+
+  context '#search' do
+    # Search falls outside of the normal testing because it is not exactly
+    # RESTy. The search endpoint (eventually) can return any type of resource
+
+    include_context 'client'
+
+    before do
+      # search is not supported on the free tier, except in the sandbox. In
+      # order for VCR to record the response, we need the endpoint to work
+      client.endpoint = 'https://sandbox.iexapis.com/v1'
+      # using the documentation-provided token
+      client.publishable_token = 'Tsk_341c973d296e4e18aa61dd63050ce235'
```

Comment:
We default this to `token` and replace it everywhere for tests, as they are run from a recording. Don't leak your tokens!

Reply:
This particular endpoint is not available to free tier accounts (the plan I'm on!), so I had to update the endpoint to the sandbox, where of course, different keys are needed. the token in this code is copy/pasted from the iexcloud docs, so it's already publicly exposed!

Reply:
Is it that they have it in sandbox but not in production yet? In which case you can manually remove the sandbox part from tests and edit VCRs once you have a working recording.